lldb/source/Core/ModuleList.cpp

```diff
@@ -136,6 +136,11 @@ bool ModuleListProperties::GetUseSwiftDWARFImporter() const {
   return m_collection_sp->GetPropertyAtIndexAsBoolean(
       NULL, idx, g_modulelist_properties[idx].default_uint_value != 0);
 }
+
+bool ModuleListProperties::SetUseSwiftDWARFImporter(bool new_value) {
+    return m_collection_sp->SetPropertyAtIndexAsBoolean(
```

Comment:
Nit, but this doesn't seem to be clang-formatted. Can you run `clang-format` on your patch?

Reply:
Hi @dcci! Is the clang-format style LLVM? `clang-format` doesn't change this particular method (it does change some other lines in the file though).

Reply:
If `clang-format` doesn't change anything, then there's nothing to do here.